### PR TITLE
`Popup:` Adds FieldsPopupElement composable

### PR DESCRIPTION
**Pull request by eri9000**
Adds FieldsPopupElement composable to display FieldsPopupElements in a Popup.

### CODE REVIEW

**Review by sorenoid — APPROVED** [review]

File: toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/internal/element/fieldselement/FieldsPopupElement.kt [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+ *
+ * @since 200.5.0
+ */
+@Composable
+internal fun FieldsPopupElement(state: FieldsElementState) {
+    val localContext = LocalContext.current
+    ExpandableCard(
+        title = state.title,
+        description = state.description,
+        elementCount = state.fieldsToFormattedValues.size,
```
Comment (shubham7109) [inline toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/internal/element/fieldselement/FieldsPopupElement.kt:49]: This looks to be using `elementCount` which has been removed from `ExpandableCard` in this PR. I was planning on reviewing it when I reached this compilation error.

**Comment by puneet-pdx** [comment]
@eri9000 this branch also has conflicts, can you please resolve them.

**Review by puneet-pdx — COMMENTED** [review]
@eri9000 this branch also has conflicts, can you please resolve them.

**Review by puneet-pdx — APPROVED** [review]
@eri9000 changes look good, but I do see a flicker when scrolling the fields popup element and it seems the list resets and I have to scroll twice as much to get back to the top.

**Pull request merged**